Docs/Monogame.Extended.Tiled/TiledMap.md

```diff
@@ -0,0 +1,69 @@
+<a alight="right" href="https://www.mapeditor.org/"><img align="right" src="https://raw.githubusercontent.com/bjorn/tiled/master/src/tiled/images/about-tiled-logo.png" height="200"></a>
+
+# Tiled
+Tiled is an [open sourced](https://github.com/bjorn/tiled) free to use "generic tile map editor". Tiled lets you easily design and view tile maps, and through the `Monogame.Extended.Tiled` package, you can load and display a map generated with Tiled in monogame
+
+To load a TiledMap into your project, you first need to add it to your `ContentManager`, and in order to compile the tile map for use in `MonoGame` you must add a reference to the pipeline tool.
+
+To do so, first open the `Content.mgcb` file in the `Content/` folder
+
+![Imgur](https://i.imgur.com/fvHuZcu.png)
+
+Add a reference to `MonoGame.Extended.Content.Pipeline.dll` in the `packages/MonoGame.Extended.Content.Pipeline.X.X.X/tools/` directory
```

Comment:
Perhaps we don't need to explain this part and instead we can just reference the documentation for adding the reference itself?

Reply:
we could, I didn't even know there was a section that described this

Reply:
@DusterTheFirst Yeah, heres the section: http://craftworkgames.github.io/MonoGame.Extended/installation/